yt/visualization/volume_rendering/scene.py

```diff
@@ -246,7 +247,12 @@ def _sanitize_render(self, render):
 
         return render
 
-    def save(self, fname=None, sigma_clip=None, render=True):
+    def save(
+        self,
+        fname: Optional[str] = None,
+        sigma_clip: Optional[int, float] = None,
+        render: bool = True,
```

Comment:
and here

Reply:
253 was the line causing the failure, it is fixed now